### DISCUSSION TITLE
How to access a static file in public/subfolder?

### DISCUSSION THREAD

**Discussion by crazyyi** [i18next/next-i18next#2132] [General]
For example,
I have a pdf file inside `./src/public/documents` and I can access it using "localhost:3000/documents/file.pdf",
But if I switch to a new locale, The URL becomes "localhost:3000/es/documents/file.pdf",
then I am unable to open the file.
I want to have different files for different locales. But the problem I am facing now is that I can't even access the subfolder `documents` in other locales. 
Can I config that or do I need to write some code for that purpose?

**Comment by adrai** [comment]
Ask this in the vercel/next.js repo.
